cli/fix/typescript/typescript.go

```diff
@@ -0,0 +1,272 @@
+package typescript
+
+import (
+	"encoding/json"
+	"flag"
+	"io/ioutil"
+	"log"
+	"path"
+	"regexp"
+	"sort"
+	"strings"
+
+	"github.com/bazelbuild/bazel-gazelle/config"
+	"github.com/bazelbuild/bazel-gazelle/label"
+	"github.com/bazelbuild/bazel-gazelle/language"
+	"github.com/bazelbuild/bazel-gazelle/repo"
+	"github.com/bazelbuild/bazel-gazelle/resolve"
+	"github.com/bazelbuild/bazel-gazelle/rule"
+)
+
+const (
+	languageName        = "ts"
+	tsProjectRuleName   = "ts_project"
```

Comment:
does this work with our `ts_library` macro?

Reply:
You have to set this (which I'll add to our BUILD file when I send out the PR to run this over our repo) - which works: 
```
# gazelle:map_kind ts_project ts_library //rules/typescript:index.bzl
```